ONBOARDING.md

```diff
@@ -1,6 +1,10 @@
 # Onboarding
 
-Setting up the GnosisVPN PoC can be somewhat complex, as it involves multiple steps and configuration details:
+Setting up the GnosisVPN PoC can be somewhat complex, as it was conceived as a technical show case and not a full-fledged product. If you are not comfortable with editing configuration files and using your terminal, please wait for the next version, which will have a streamlined user experience.
```

Comment:
Setting up the GnosisVPN PoC can be somewhat complex, as it was designed as a technical showcase rather than a full-fledged product. If you are not comfortable editing configuration files or using your terminal, please wait for the next version, which will offer a more streamlined user experience.